ourjapanlife/findadoc-web
# feat: show modal to confirm submission rejection

Resolves #770 

## 🔧 What changed

1. A modal will now show when the user clicks the "Reject" button. Clicking the confirmation button on the modal will reject the submission and navigate the user back to the moderation screen.
2. The div that was wrapped around the modal was removed, and its classes were moved to the `<Modal>` component.

## TODO
We should probably consider fetching the submissions from the database again once the user clicks the rejection button. Otherwise, it won't appear under Rejected until the page is refreshed.

## 🧪 Testing instructions
1. On your local dev server, select a submission and click the Reject button.
2. Click the X button to close the modal without rejecting the submission.
4. Click the confirmation button on the modal to reject the submission and return to the moderation screen.
5. Refresh the page to confirm that the submission is now rejected.

## 📸 Demo
https://github.com/user-attachments/assets/8b60c892-04f7-4116-a250-72cc9ab9308f
